microsoft-edge/devtools-guide-chromium/index.md

```diff
@@ -3,7 +3,7 @@ description: Get to know the Microsoft Edge (Chromium) Developer Tools
 title: Microsoft Edge (Chromium) Developer Tools overview  
```

Comment:
```suggestion
title: Microsoft Edge Developer Tools overview  
```
AND change # h1 top-of-page title to match.  
AND in first sentence of body, maybe change:
When you install Microsoft Edge
to:
When you install Microsoft Edge (Chromium)

Reply:
Done